pkg/webui/console/views/application-add/index.js

```diff
@@ -99,7 +99,7 @@ export default class Add extends React.Component {
   }
 
   @bind
-  async handleSubmit(values, { resetForm }) {
+  async handleSubmit(values, { resetForm, setSubmitting }) {
```

Comment:
`resetForm` is not used
```suggestion
  async handleSubmit(values, { setSubmitting }) {
```